### ISSUE TITLE
deadline to choose a single pair of models for internal review draft?

### ISSUE THREAD

**Issue by iantaylor-NOAA** [iantaylor-NOAA/Lingcod_2021#77]
It's easy to keep tinkering with the model for a long time, but if we want the internal review draft to represent a single pair of models, and have the associated profiles and retrospectives be from that same pair of models, we should figure out how long it takes to reproduce all the associated stuff from these models and count backwards from the deadline to figure out when we need to finalize the choice.

Alternatively we could submit an internal review report that is internally inconsistent but include the most recently produced figures and tables for each element.

I don't have any strong preference on this.

**Comment by kellijohnson-NOAA** [comment]
I started the profiles around 11 pm I think and they were not done until mid morning. I did not run retrospectives or jitter. I think we could start a jitter on any model and it would be helpful, to know if it does or doesn't get back to the MLE. I will actually start them right now. But wrt to base models. If there are more things you want to explore, I think it would be fine to start profiles and retro tomorrow.
